processing/src/main/java/org/apache/druid/query/DefaultQueryMetrics.java

```diff
@@ -243,6 +243,12 @@ public void identity(String identity)
     return reportMillisTimeMetric("query/node/ttfb", timeNs);
   }
 
+  @Override
+  public QueryMetrics<QueryType> reportBackPressureTime(long timeNs)
+  {
+    return reportMillisTimeMetric("query/node/backpressure", timeNs);
```

Comment:
This should be empty and overridden with actual emit in a private extension. Or, an alternative set of "full" QueryMetrics implementations should be provided, as suggested here: https://github.com/apache/incubator-druid/pull/4570#issuecomment-321026206

Reply:
Read the Javadoc for `QueryMetrics`.